drivers/ve-direct.c

```diff
@@ -0,0 +1,298 @@
+/* ve-direct.c - Model specific routines for Victron Energy Direct units
+ *
+ * Copyright (C) 2017  Petr Kubanek <kubanek@fzu.cz>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ *
+ */
+
+#include "main.h"
+#include "serial.h"
+
+#define DRIVER_NAME	"Victron Energy Direct UPS and solar controller driver"
+#define DRIVER_VERSION	"0.20"
```

Comment:
GiHub web interface doesn't quite let decipher - is there a whitespace between macro name and value (of driver version)?